src/app/api/gql/scheduling-dataset/set-config-batching.graphql

```diff
@@ -0,0 +1,40 @@
+mutation setConfigBatching(
```

Comment:
Name is bad. No mentioning of a flow, no mentioning of a dataset.

Reply:
Renamed to `datasetFlowBatching`